README.md

```diff
@@ -480,12 +480,11 @@ To add Mantle to your application:
  1. Run `script/bootstrap` from within the Mantle folder.
  1. Drag and drop `Mantle.xcodeproj` into your application's Xcode project or
     workspace.
- 1. On the "Build Phases" tab of your application target, add Mantle to the
-    "Link Binary With Libraries" phase.
-    * **On iOS**, add `libMantle.a`.
-    * **On OS X**, add `Mantle.framework`. Mantle must also be added to any
-      "Copy Frameworks" build phase. If you don't already have one, simply add a
-      "Copy Files" build phase and target the "Frameworks" destination.
+ 1. On the "General" tab of your application target, link Mantle.
+    * **On iOS**, add `Mantle.framework` to the "Embedded Binaries" build phase.
```

Comment:
It's not a build phase, but just a pane on the General tab. Can you rephrase that slightly?